crypto/KeyDialog.cpp

```diff
@@ -359,7 +359,7 @@ void CertModel::load( const QList<QSslCertificate> &result )
 	{
 		SslCertificate c( k );
 		if( c.keyUsage().contains( SslCertificate::KeyEncipherment ) &&
-			!c.enhancedKeyUsage().contains( SslCertificate::ServerAuth ) &&
+			c.enhancedKeyUsage().contains( SslCertificate::All ) &&
```

Comment:
Because SslCertificate::enhancedKeyUsage() - see https://github.com/open-eid/qt-common/blob/master/SslCertificate.cpp#L75 - sets the key to All if certificate has no enhanced key usage.